### PR TITLE
Rmoving duplicate test-configuration for "folder traversal spec not recursing fully"

### PR DESCRIPTION
**Pull request by yaacov**
Issue:
similar tests run on tier1 and wmware tests.
the issue tested in tier1 is a backend bug, and it will be tested again in vmware section.
the config files for the tests are identical.

suggestion:
add a note explaining why we use config file from vmware folder, and reuse the same config file

Signed-off-by: yaacov <kobi.zamir@gmail.com>

### CODE REVIEW

**Comment by yaacov** [comment]
@mturley @ibragins please review

**Review by mturley — APPROVED** [review]
LGTM, will wait for @ibragins approval before merging

**Review by ibragins — APPROVED** [review]
LGTM, it can be merged

**Pull request merged**
